exercises/rational-numbers/canonical-data.json

```diff
@@ -26,7 +26,7 @@
               "expected": [7, 6]
             },
             {
-              "description": "Add a positive rational number and a negative rational number",
```

Comment:
This distinction is necessary because it's also possible to add a positive number, e.g. 6.348 or π, to rational numbers.